crates/bevy_ecs/README.md

```diff
@@ -1,7 +1,3 @@
 # Bevy ECS
 
-Bevy Engine's entity component system
-
-## Licensing
-
-Bevy ECS has its roots in hecs, which is licensed as Apache 2.0. All original hecs code is licensed under Apache 2.0. All added/modified code is dual licensed under MIT and Apache 2.0. Files with an Apache 2.0 license header (with Google LLC as the copyright holder) were from the "original hecs" codebase. Files without the header were created by Bevy contributors.
```

Comment:
I dont think you shoulf get rid of this as its still licensed, just under MIT

Reply:
Thats a good point. I'll re-add something along those lines if/when we get the go-ahead from the lawyers.